readme.txt

```diff
@@ -1,5 +1,5 @@
 === Remote Data Blocks ===
-Contributors: chriszarate, maxschmeling, mhsdef and others.
+Contributors: czarate, maxschmeling, hew.
```

Comment:
Shouldn't this include the other members of the team as well?

Reply:
Not the PR author, but I think the purpose of this PR is just to fix the list as it exists today. This is a project with many contributors, so the question is how many of them get added here, and which are the criteria of addition. I think that having a common convention about this across our plugins could be useful.

Reply:
> Shouldn't this include the other members of the team as well?

It could do, but like @acicovic says, it was to fix an existing problem (.org usernames were invalid). Most probably we want to set some criteria for inclusion into this block (50+ commits or notable contributions?).

Reply:
Yeah, fixing the *issue* is good. Would like to determine the "process" for this. Anyone have context on how plugins typically do it? Should it be the *active* contributors? All time contributors with a certain criteria like @pkevan suggested?

Reply:
> Yeah, fixing the _issue_ is good. Would like to determine the "process" for this. Anyone have context on how plugins typically do it? Should it be the _active_ contributors? All time contributors with a certain criteria like @pkevan suggested?

Can be however we want, but usually it just sticks the same like Gutenberg since no-one really has a process.